doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -834,6 +834,33 @@ have the following different capability levels:
 -  Auto pilot: Horizontal/vertical scaling, automated configuration tuning, abnormality detection, and scheduling
    tuning.
 
+Platform services
+-----------------
+
+The archirecture may support functionalities on top of the infrastructure services. With Kubernetes and the cloud native
+paradigms the demarcation between infrastructure services and platform services are fuzzy. In this specification we
+considider every service what is provided by Kuberntes with the help of supporting components as infrastructure service,
+while all services provided by independent open source components are considered as platform services. Based on the
+specification of Chapter 5.1.5 of the Reference Model (RM) :cite:p:`refmodel` the following platform services may be
+supported by an architecture:
+
+- Data stores/databases
+- Streaming and messaging
+- Load balancer and service proxy
+- Service mesh
+- Security and compliance
+- Monitoring
+- Logging
+- Application definition and image build
+- CI/CD
+- Ingress/egress controllers
+- Network service
```

Comment:
is this one CNI? or ETSI NS? or something else?

Reply:
RM Table 5-7 brings CoreDNS and Istio as examples. What if I write "Network related services" here?

Reply:
Added "Network related services" in https://github.com/anuket-project/RA2/pull/8/commits/ff3919c849380eba051621a2fdf5c90ae1958627 and proposed a fix to RM in https://github.com/anuket-project/RM/pull/20